src/components/PersonString/PersonString.tsx

```diff
@@ -0,0 +1,50 @@
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+interface Props {
+  person: Person;
+  personSlug: string;
+}
+
+export const PersonString:React.FC<Props> = ({ person, personSlug }) => {
+  const {
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = person;
+
+  const isSelected = personSlug === slug;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': isSelected,
+      })}
+    >
+      <td>
+        <PersonLink person={person} />
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {mother
+          ? <PersonLink person={mother} />
+          : motherName || '-'}
```

Comment:
ternary operator should return one value depending on condition, don't make another condition in the ternary operator